libs/metamask/src/lib/+state/erc1155.ts

```diff
@@ -11,59 +11,53 @@ interface ERC1155_Event {
   URI: (value: string, id: number) => void;
 }
 
+export interface ERC1155_Meta {
+  name: string;
+  description: string;
+  image: string;
+  image_data?: string;
+  background_color?: string;
+  animation_url?: string;
+  youtube_url?: string;
+  attributes: { display_type?: string, trait_type: string, value: string }[];
+}
+
+export interface ERC1155_Token {
+  id: string;
+  account: string;
+  balance: number;
+  meta: ERC1155_Meta;
+}
 
 @Injectable({ providedIn: 'root' })
 export class ERC1155 extends Contract {
-
   constructor(metamask: MetamaskService) {
     if (!metamask.signer) throw new Error('ERC1155 requires a signer');
     super(env.eth.erc1155, abi, metamask.signer);
   }
 
-  on<K extends keyof ERC1155_Event>(event: K, listener: ERC1155_Event[K]) {
-    return super.on(event, listener)
-  }
-  off<K extends keyof ERC1155_Event>(event: K, listener: ERC1155_Event[K]) {
-    return super.off(event, listener)
-  }
-  once<K extends keyof ERC1155_Event>(event: K, listener: ERC1155_Event[K]) {
-    return super.once(event, listener)
-  }
-  removeAllListeners(event?: Extract<keyof ERC1155_Event, string> | EventFilter) {
-    return super.removeAllListeners(event);
-  }
-  
-  /**
-    Transfers `value` amount of an `id` from the `from` address to the `to` address specified (with safety call).
-    @param from    Source address
-    @param to      Target address
-    @param id      ID of the token type
-    @param value   Transfer amount
-    @param data    Additional data with no specified format, MUST be sent unaltered in call to `onERC1155Received` on `to`
-  */
-  safeTransferFrom(from: string, to: string, id: number, value: number, data: string): Promise<Transaction> {
-    return super.safeBatchTransferFrom(from, to, id, value, data);
+  async getTokens(account: string) {
```

Comment:
can you add return type ?

Reply:
It's generated automatically by Typescript